src/main/scala/Benchmarks/TriesBenchmark.scala

```diff
@@ -0,0 +1,82 @@
+package Benchmarks
```

Comment:
Maybe the whole file should be removed from pr and refactored later?

Reply:
Maybe it should be moved to tests?